maplibre/src/io/geometry_index.rs

```diff
@@ -62,6 +62,12 @@ impl GeometryIndex {
     }
 }
 
+impl Default for GeometryIndex {
+    fn default() -> Self {
+        Self::new()
+    }
+}
```

Comment:
I was wondering which was is better: 
* Implement the constructor in new()
* or implement it in default() and then call default() in new()

What do you think?

Reply:
No strong feeling and I'm not a Rust expert. But in the [clippy docs](https://rust-lang.github.io/rust-clippy/master/#new_without_default) they seem to point ot this pattern (just have default delegate to new)

Reply:
Oke cool, that let's use that style!